src/Employer/Employer.Web/ViewModels/Part1/EmployerName/EmployerNameOptionViewModel.cs

```diff
@@ -6,6 +6,7 @@ public enum EmployerNameOptionViewModel
     {
         RegisteredName,
```

Comment:
Can this be renamed to `EmployerIdentityOption` as it's slightly different to domain `EmployerNameOption` and also this isn't a ViewModel. Would suggest putting this in the Employer Web Models folder/namespace. `EmployerInfoModel` uses it so it is wrong that Models are referencing classes defined in the ViewModels namespace.

Reply:
👍